agreements/project-contribution/asset-transfer-amendment-email.md

```diff
@@ -0,0 +1,51 @@
+# Email Template for Amendment to Asset Transfer Agreement
+
+This email format can be used by a Project Representative or Signatory to request a change to the Asset Transfer Agreement.
+
+An amendment is valid if:
+
+- All required parties are included on the thread (e.g. Commonhaus, the Assignor)
+- The email clearly states what is being amended
+- Each party replies affirmatively to confirm
+
+Notes:
+
+- `project-[project-name]@commonhaus.org` is the shared group email provided by Commonhaus for communication with project signatories.
```

Comment:
I actually just asked about that in Discord - what process creates this address?

Reply:
magic on-boarding machinery (i.e. scripts)